pmxbot/core.py

```diff
@@ -430,9 +430,9 @@ def merge_dicts(a, b):
 
 
 class Bot(metaclass=abc.ABCMeta):
-	def out(self, channel, s, log=True):
+	def out(self, channel, s, log=True, context=None):
 		try:
-			sent = self.allow(channel, s) and self.transmit(channel, s)
+			sent = self.allow(channel, s) and self.transmit(channel, s, context)
```

Comment:
Yes, this should probably be breaking things since I didn't change the signature on `core.Bot.transmit` or `irc.LoggingCommandBot.transmit`. I was a bit unsure about testing actually; I can't tell if this code is being executed in the existing tests?